lib/cops/readme.md

```diff
@@ -19,3 +19,39 @@ Enabling the cop inside .rubocop.yml :
 CustomCop/PutLoggerFormatCop:
   Enabled: true
 ~~~
+
+### DuplicateConstantCop
+DuplicateConstantCop ensures that there is no two constants in a file which has same string value
```

Comment:
can you add an example of why it is not recommented to have the same value for constants?